__tests__/app.test.js

```diff
@@ -25,3 +25,31 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET generic not found url request", () => {
+  test("404: Responds with Not found when url does not exist", () => {
+    return request(app)
+      .get("/api/bananas")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+});
+
```

Comment:
Good work with this error test, it belongs in it's own describe block as it's separate from any other endpoint!